MediaBrowser.Controller/Lyrics/Lyric.cs

```diff
@@ -0,0 +1,28 @@
+namespace MediaBrowser.Controller.Lyrics;
+
+/// <summary>
+/// Lyric model.
+/// </summary>
+public class Lyric
```

Comment:
I think calling this "LyricLine" might make more sense because it's the information for a single line in a lyrics file.